cmd/kubeadm/app/master/apiclient.go

```diff
@@ -55,20 +55,11 @@ func CreateClientAndWaitForAPI(file string) (*clientset.Clientset, error) {
 		if len(nodeList.Items) < 1 {
 			return false, nil
 		}
-		n := &nodeList.Items[0]
-		if !v1.IsNodeReady(n) {
-			fmt.Println("[apiclient] First node has registered, but is not ready yet")
-			return false, nil
-		}
 
-		fmt.Printf("[apiclient] First node is ready after %f seconds\n", time.Since(start).Seconds())
+		fmt.Printf("[apiclient] First node has registered after %f seconds\n", time.Since(start).Seconds())
 		return true, nil
 	})
 
-	if err := createAndWaitForADummyDeployment(client); err != nil {
```

Comment:
I suggest we move this and the node ready check to a seperate final validation step. Do you mean we should drop it permanently?